src/activities/act-7-5-1.xml

```diff
@@ -77,21 +77,118 @@
   </hint>
   <answer>
     <p>
-      <ol label="a.">
+      <ol label = "a.">
+        <li>
+          <p>
+            <m>\frac{dA}{dt} = 0.05A</m>.
+          </p>
+        </li>
+
+        <li>
+          <p>
+            <m>
+              \frac{dA}{dt} = 0.05A - 10000.
+            </m>
+          </p>
+        </li>
+
+        <li>
+          <p>
+              <figure xml:id="F-act-7-5-1-a">
+                <caption>Slope field for <m>\frac{dA}{dt} = 0.05A - 10000</m>.</caption>
+                <image source="images/7_5_Act1_slope_field">
+                </image>
+              </figure>
+            The only equilibriume solution is <m>A = 200000</m>.  
+          </p>
+        </li>
+
+        <li>
+          <p>
+            <m>t = 20 \ln(2) \approx 13.86</m>years.
+          </p>
+        </li>
+
+        <li>
+          <p>
+            At least <dollar />200000.
+          </p>
+        </li>
+
         <li>
           <p>
-            Answers for each of the prompts above.
+            Up to <dollar />15000 every year.
           </p>
         </li>
       </ol>
     </p>
   </answer>
   <solution>
     <p>
-      <ol label="a.">
+      Let <m>A(t)</m> be the amount of money in a bank account in year <m>t</m> that grows by 5<percent /> every year.
+      <ol label = "a.">
+        <li>
+          <p>
+            The rate of change of <m>A</m> with respect to <m>t</m> is <m>\frac{dA}{dt} = 0.05A</m>.
+          </p>
+        </li>
+
+        <li>
+          <p>
+            If <dollar />10000 is being withdrawn every year, then
+            <me>
+              \frac{dA}{dt} = 0.05A - 10000.
+            </me>
+          </p>
+        </li>
+
+        <li>
+          <p>
+            Following is a slope field for the differential equation in (b).
+              <figure xml:id="F-act-7-5-1-s">
+                <caption>Slope field for <m>\frac{dA}{dt} = 0.05A - 10000</m>.</caption>
+                <image source="images/7_5_Act1_slope_field">
```

Comment:
Should this refer to `images/7_5_slope_field`? I don't see a `7_5_Act1_slope_field`